front/pages/w/[wId]/developers/api-keys.tsx

```diff
@@ -0,0 +1,69 @@
+import { Page, ShapesIcon } from "@dust-tt/sparkle";
+import type { GroupType, UserType, WorkspaceType } from "@dust-tt/types";
+import type { SubscriptionType } from "@dust-tt/types";
+import type { InferGetServerSidePropsType } from "next";
+import React from "react";
+
+import { subNavigationAdmin } from "@app/components/navigation/config";
+import AppLayout from "@app/components/sparkle/AppLayout";
+import { withDefaultUserAuthRequirements } from "@app/lib/iam/session";
+import { APIKeys } from "@app/pages/w/[wId]/a";
+
+const { GA_TRACKING_ID = "" } = process.env;
```

Comment:
This should be retrieved using the config helper `config.getGaTrackingId()`.

Reply:
OK, I replace both occurences in my PR but I'll make another PR to fix them all. That should be an easy refactor we should not wait.

Reply:
https://github.com/dust-tt/dust/pull/7024

Reply:
Gracias 🙏 !